aten/src/ATen/core/builtin_function.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/function.h>
+
+namespace torch {
+namespace jit {
+
+struct BuiltinOpFunction : public Function {
+  BuiltinOpFunction(c10::QualifiedName qualname, c10::Symbol op_symbol)
+      : name_(std::move(qualname)), symbol_(std::move(op_symbol)) {}
+
+  bool isGraphFunction() const override {
+    return false;
+  }
+
+  void run(Stack& stack) override {
+    auto handle = c10::Dispatcher::singleton().findSchemaOrThrow(
```

Comment:
BuiltinOp should just store its Schema, and std::function<void(Stack&)> directly. It does not have to participate in dispatch (and in fact will mostly only incur overhead because of it).

Reply:
@zdevito - hm, are you sure about that? I'd at least want to have it go through dispatcher to get profiling working. TorchBind ops can be potentially big so it's good to capture them through the same profiling infra. Alternatively - we can just slap the same RecordFunction here too

Also I'm pretty sure someone will want tracing to work too (though it's debatable)